src/ui/components/shared/Login/Login.tsx

```diff
@@ -177,21 +177,37 @@ function AuthError({ error }: { error: any }) {
     return null;
   }
 
-  if (message === "Invalid state") {
-    // This is usually caused by waiting too long to go through the auth process
-    // and can be fixed by trying again.
-    message = "Your login session expired. Please try logging in again.";
-  } else {
-    // We want to capture any other error so we can investigate further.
+  if (message !== "INVALID_STATE" && message !== "Invalid state") {
     sendTelemetryEvent("devtools-auth-error-login", {
       errorMessage: message,
     });
+  }
 
-    if (message === "Unable to authenticate user") {
+  // Except of "Invalid state" message that comes from
+  // https://auth0.com/docs/customize/integrations/cms/wordpress-plugin/troubleshoot-wordpress-plugin-invalid-state-errors
+  // we define all the other "message"s in our Auth0 Rule.
+  // See more here: https://auth0.com/docs/customize/rules/raise-errors-from-rules
+  switch (message) {
+    case "INVALID_STATE":
+    case "Invalid state":
+      // This is usually caused by waiting too long to go through the auth process
+      // and can be fixed by trying again.
+      message = "Your login session expired. Please try logging in again.";
+      break;
+    case "INTERNAL_ERROR":
       // This usually occurs because our auth hook threw an error but the
       // message itself isn't very useful so we show a more friendly message
       message = "We're sorry but we had a problem authenticating you. We're looking into it now!";
-    }
+      break;
+    case "IDP_CONFIGURATION_ERROR":
+      message =
+        "Failed to log in due to a configuration problem with your Replay account. Support has been notified!";
+    case "IDP_UNEXPECTED_ERROR":
+      message = "Failed to login. Please try logging in with SSO.";
+      break;
+    default:
+      // We want to capture any other error so we can investigate further.
```

Comment:
Should we be capturing anything here (in Sentry) btw?

Reply:
The comment was misleading. I forgot to move that up where we are capturing telemetry events, for which Ryan gets an alert.

Also from Ryan:
> I don't think it's helpful in Sentry because the cause of the error isn't in devtools. Had we sentry integration in Auth0, it might be more useful there but I think telemetry is the best we can do right now.